app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -0,0 +1,37 @@
+<head>
+  <style>
+    p {
+      font-family: Helvetica
+    }
+    .text-center{
+      text-align: center
+    }
+
+    .border-blue {
+      border-width: 1px;
+      border-color: #0F58B7;
+      border-radius: 50px;
+      border-style: solid
+    }
+
+    .text-decoration-none {
+      text-decoration: none;
+
+    }
+
+    .text-blue {
+      color: #0F58B7
+    }
+
+    .fw-600 {
+      font-weight:600
+    }
+  </style>
+</head>
```

Comment:
This should be inside of the mailer layout. Also we're adding a `head` tag in the middle of the body right here.